### ISSUE TITLE
Add support for input from STDIN

### ISSUE THREAD

**Issue by conatus** [ddksr/cliist#11]
A feature request I may well just do myself.

Accept input from STDIN a list of tasks, delimitated by a newline. cliist then adds each one as a new task.

This would allow me to add a whole list of tasks at the start of the day in the comfort of my editor, a little like Taskpaper allows.

**Comment by ddksr** [comment]
yes, this could really be helpful. could we define the workflow a bit more?
I also suggest that any `#project` or `@label` are automatically interpreted. I'm just not sure what the defaults should be. Inbox & today?
